### PR TITLE
Run2-hcx152 Test the constituent map of CaloTower

### PR DESCRIPTION
**Pull request by bsunanda**
May be used to study MIP signal in HE which is not covered by tracker

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

File: Geometry/CaloTopology/test/CaloTowerMapTester.cc [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#include "Geometry/CaloTopology/interface/CaloTowerConstituentsMap.h"
+#include "DataFormats/CaloTowers/interface/CaloTowerDetId.h"
+#include "DataFormats/EcalDetId/interface/EBDetId.h"
+#include "DataFormats/EcalDetId/interface/EEDetId.h"
+#include "DataFormats/EcalDetId/interface/ESDetId.h"
+#include "DataFormats/EcalDetId/interface/EcalSubdetector.h"
+#include "DataFormats/HcalDetId/interface/HcalDetId.h"
+#include "DataFormats/HcalDetId/interface/HcalSubdetector.h"
+
+class CaloTowerMapTester : public edm::EDAnalyzer {
```
Comment (davidlange6) [inline Geometry/CaloTopology/test/CaloTowerMapTester.cc]: hi @bsunanda - please use a non-legacy module type.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**